Htc closures (#31956)
idaholab/moose: modules/subchannel/doc/content/source/problems/TriSubChannel1PhaseProblem.md

This class solves for the subchannel flow variables in the case of subchannels/pins arranged in a triangular lattice.
It inherits from the base class : `SubChannel1PhaseProblem`. Information regarding the solver can be found in [subchannel_theory.md].

> Pin surface temperature is calculated at the end of the solve using a user defined correlation.

Reviewer: ```suggestion
Pin surface temperature is calculated at the end of the solve using a user-defined correlation.
```

> Pin surface temperature is calculated at the end of the SCM solve using a user-defined correlation.

Reviewer: ```suggestion
Pin surface temperature is calculated at the end of the SCM solve using a user-selected correlation.
```

\end{equation}

where:

- $T_{s,d}(z)$ is the duct surface temperature at a height $z$
- $T_{bulk,d}(z)$ is the bulk temperature of the subchannel next to the duct node $d$
- $q''_d(z)$ is the heat flux at the duct at a height $z$

> - $h_d(z)$ is the convective heat transfer coefficientfor the subchannel next to the duct node at a height $z$

Reviewer 2: Small typo here: coefficientfor instead of coefficient for

- $k$ is the thermal conductivity of the subchannel neighboring the structure
- $D_h$ is the hydraulics diameter of the subchannel neighboring the structure

> The modeling of the Nusselt number and consequently of the heat transfer coefficient `h` is defined by the user. The closure models available to the user that are implemented in SCM are the following:

Reviewer: ```suggestion
The modeling of the Nusselt number and consequently of the heat transfer coefficient `h` is selected by the user through a closure. The closure models available to the user that are implemented in SCM are the following:
```

- $k$ is the thermal conductivity of the subchannel neighboring the structure
- $D_h$ is the hydraulics diameter of the subchannel neighboring the structure

The modeling of the Nusselt number and consequently of the convective heat transfer coefficient `h` is selected by the user through a closure. The closure models available to the user that are implemented in SCM are the following:

- [Dittus-Boelter](SCMHTCDittusBoelter.md) (recommended for water coolants)
- [Gnielinski](SCMHTCGnielinski.md) (recommended for liquid metals)
- [Kazimi-Carelli](SCMHTCKazimiCarelli.md) (recommended for liquid metals)

> 

Reviewer: same update the list
